orttraining/orttraining/python/orttraining_pybind_state.cc

```diff
@@ -9,6 +9,9 @@
 
 #include "core/session/environment.h"
 #include "orttraining/core/session/training_session.h"
+#if !defined(ORT_MINIMAL_BUILD)
```

Comment:
we don't need this line either. 

MINIMAL_Build is only an option for inference-only build. 
with enable_training = true, we should ignore MINIMAL_Build setting.